source/controllers/index.md

```diff
@@ -1,14 +1,23 @@
 ## Controllers
 
-In Ember.js, controllers allow you to decorate your models with
-display logic. In general, your models will have properties that
-are saved to the server, while controllers will have properties
-that your app does not need to save to the server.
+Modern Ember applications don't often use controllers. When they do, their
+responsibility is strictly limited to two avenues:
 
-For example, if you were building a blog, you would have a
-`BlogPost` model that you would present in a `blog_post` template.
+* Controllers maintain state based on the current route. In general, models
```

Comment:
@locks The key phrase here is "state _based on the current route_". Services are global, and we don't yet have a great place to put state that is "global" to a route but not the wider application. /cc @wycats